catchup/service.go

```diff
@@ -428,9 +434,16 @@ func (s *Service) fetchAndWrite(ctx context.Context, r basics.Round, prevFetchCo
 
 // TODO the following code does not handle the following case: seedLookback upgrades during fetch
 func (s *Service) pipelinedFetch(seedLookback uint64) {
```

Comment:
It wasn't clear to me why the seedLookback is a minimum. It is 2.

Reply:
I guess because a node requires seedLookback blocks back to operate normally so fetching at least these two blocks.